src/commands.ts

```diff
@@ -226,13 +226,22 @@ async function readOrRequestFileExtensionConfig(uri: vscode.Uri): Promise<string
     }
 
     if (!extensions) {
-        extensions = await vscu.showInputBox(
-            'Please enter the file extensions your DataPlugin can handle in the right syntax: ',
-            '*.tdm; *.xls ...'
-        );
+        // eslint-disable-next-line no-constant-condition
+        while (true) {
+            // eslint-disable-next-line no-await-in-loop
+            extensions = await vscu.showInputBox(
+                'Please enter the file extensions your DataPlugin can handle in the right syntax: ',
```

Comment:
Reading the text, it's not really clear to me that you can enter multiple extensions. And the question "What is the right syntax?" may occur. What do you think about

`Please enter one or multiple file extensions your DataPlugin can handle in the shown format: `

Or maybe @GrKuehnH has a better idea.

Reply:
The placeholder text should give a hint what the right syntax looks like. See screenshot that I attached to the PR. I'm okay with your text proposal. Maybe we find something better for "can handle". @GrKuehnH thanks!

Reply:
Suggestion: Enter the file extension(s) your DataPlugin handles using the following syntax:

I do remember, however, that we were told some years ago that we weren't supposed to use the asterisk when writing filenames in documentation: .xml; .tdm.  Does this also apply to software?

Reply:
I stuck with the rules that DIAdem specified for the file extension input:

![image](https://user-images.githubusercontent.com/79531/121482724-afcf2900-c9cd-11eb-9552-f7a2970ace87.png)

I like your suggestion @GrKuehnH !